tests/testVisitInfo.py

```diff
@@ -195,10 +195,10 @@ def testSetVisitInfoMetadata(self):
 
     def testSetVisitInfoMetadataMissingValues(self):
         """If a value is unknown then it should not be written to the metadata"""
-        visitInfo = afwImage.makeVisitInfo()  # only rot type is known
+        visitInfo = afwImage.VisitInfo()  # only rot type is known
```

Comment:
If we don't make this change then the test will give deprecation warnings. Your call but I would like your opinion given the warnings.